app/types/actions/windowActions.ts

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as types from '../constants/windowTypes'
+
+// TODO @bbondy what else do we have inhere?
+export interface Window {
```

Comment:
You can see all properties and types here, but we only use `focused` and `id` right now.
https://developer.chrome.com/extensions/windows#type-Window

Reply:
done